src/states_screens/race_gui.cpp

```diff
@@ -82,9 +82,27 @@ RaceGUI::RaceGUI()
     else
         m_lap_width = font->getDimension(L"9/9").Width;
 
+    float map_size_splitscreen = 1.0f;
+
+    // If there are four players or more in splitscreen
+    // and the map is in a player view, scale down the map
+    if (race_manager->getNumLocalPlayers() >= 4 && !race_manager->getIfEmptyScreenSpaceExists())
+    {
+        if ( ((float) irr_driver->getFrameSize().Width / (float) irr_driver->getFrameSize().Height) >
+             (4.1f/3.0f))
```

Comment:
Where does 4.1/3 come from? That seems pretty arbitrary. That's not necessarily a showstopper, but using a constant with some explanation of where the value comes from would be preferred

Reply:
Screen ratios are mainly 16:9 and 4:3. This is a way to do "if you have 16:9 resolutions, do this, if you have 4:3 resolution, do that instead".

Reply:
Okay, why the .1 though? And a little comment explaining about screen ratios would be welcome there

Reply:
It's there in case there is some pixel off (if STK try to fit the screen in windowed mode, it will go outside of the predefined resolutions ; I haven't tested extensively how this affect the screen ratio but it won't be exactly 4:3 anymore for sure) or rounding error.